README.md

```diff
@@ -61,17 +69,16 @@ file (assuming the `erlang_ls` executable is available in your `PATH`:
     ;; Ensure docs are visible
     (setq lsp-ui-doc-enable t)
 
-    ;; Enable LSP automatically for Erlang files
-    (add-hook 'erlang-mode-hook #'lsp)
-
-    ;; TCP connection timeout
-    (setq lsp-tcp-connection-timeout 500)
-    ;; Set path to erlang_ls escript (unless it's in your path)
-    (setq lsp-erlang-server-path "/path/to/erlang_ls")
+    ;; ----- company-lsp -----
+    ;; Enables better integration with company (auto-completion)
+    (package-install 'company-lsp)
+    (push 'company-lsp company-backends)
 
-    ;; Override the key bindings from erlang-mode to use LSP for completion
```

Comment:
@jfacorro Why did you delete this? I needed it for my completion to work correctly (I also have the other parts you are adding).

Reply:
Restoring in #276